haystack/nodes/prompt/prompt_node.py

```diff
@@ -211,11 +211,13 @@ def invoke(self, *args, **kwargs):
         pass
 
     @classmethod
-    def supports(cls, model_name_or_path: str) -> bool:
+    def supports(cls, model_name_or_path: str, **kwargs) -> bool:
         """
         Checks if the given model is supported by this invocation layer.
 
         :param model_name_or_path: The name or path of the model.
+        :param kwargs: additional keyword arguments passed to the underlying model which might be used to determine
```

Comment:
```suggestion
        :param kwargs: Additional keyword arguments passed to the underlying model which might be used to determine
```

Reply:
do you think it's going to be clear to the users what they can input here? maybe we could give them some examples?
how do keywords help determine if a model is supported?

Reply:
It is not needed @agnieszka-m. Only super-user engineers need it, and they might even find the comment here unusual. PromptModelInvocationLayer, StopWordsCriteria, HFLocalInvocationLayer, OpenAIInvocationLayer, AzureOpenAIInvocationLayer should be excluded from all public docs.